From 34ec0d65ede8392ba3e0f114d72672b3f517f964 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Tue, 10 May 2016 01:09:34 +0200
Subject: [PATCH] do not pass type to std::make_pair()

We not pass type template arguments to std::make_pair()
explicitly, and let the compiler deduce them on its own.
This fixes compiler error on GCC 6.1.1.
---
 client/auth_utils.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/auth_utils.cc b/client/auth_utils.cc
index 21416bb..03fd71c 100644
--- a/client/auth_utils.cc
+++ b/client/auth_utils.cc
@@ -61,7 +61,7 @@ int parse_cnf_file(istream &sin, map<string, string > *options,
     getline(sin, option_value);
     trim(&option_value);
     if (option_name.length() > 0)
-      options->insert(make_pair<string, string >(option_name, option_value));
+      options->insert(make_pair(option_name, option_value));
   }
   return ALL_OK;
   } catch(...)
